staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -445,7 +445,7 @@ func (f *DeltaFIFO) Replace(list []interface{}, resourceVersion string) error {
 			return KeyError{item, err}
 		}
 		keys.Insert(key)
-		if err := f.queueActionLocked(Sync, item); err != nil {
+		if err := f.queueActionLocked(Replaced, item); err != nil {
```

Comment:
```
action := Sync
if f.clientUnderstandsReplacedDeltaType {
  action = Replaced
}
if err := f.queueActionLocked(action, item); err != nil {
```

Reply:
*nod*

Reply:
agree, and can set that to true in both places we call NewDeltaFIFO in-tree (in newInformer and sharedIndexInformer#Run)